### PR TITLE
[v1.7.x] Add Batch Support for Message consumer Mock

### PR DESCRIPTION
**Pull request by gajosu**
Hey @mateusjunges, support for batches was missing, I added 2 tests, one with a single batch, and another with 3 batches, I await your comments.

### CODE REVIEW

File: tests/KafkaFakeTest.php [hunk 1]
```diff
@@ -329,4 +327,137 @@ public function testStopFakeConsumer()
...
+        )
+            ->enableBatching()
+            ->withBatchSizeLimit(10)
+            ->withHandler(function (Collection $messages) use (&$consumedMessages) {
+                $consumedMessages = $messages->toArray();
+            })
+            ->build();
+
+        $consumer->consume();
+        $this->assertTrue($msgs == $consumedMessages);
```
Comment (mateusjunges) [inline tests/KafkaFakeTest.php:371]: Can't we use `assertEquals` here?

File: tests/KafkaFakeTest.php [hunk 2]
```diff
@@ -329,4 +327,137 @@ public function testStopFakeConsumer()
...
+                    $thirdBatch = $messages->toArray();
+                    $this->assertEquals(1, $messages->count());
+                }
+            })
+            ->build();
+
+        $consumer->consume();
+
+        $consumedMessages = array_merge($firstBatch, $secondBatch, $thirdBatch);
+        $this->assertTrue($msgs == $consumedMessages);
```
Comment (mateusjunges) [inline tests/KafkaFakeTest.php:460]: same here

**Review by mateusjunges — COMMENTED** [review]
Just a few minor fixes but at first glance it looks good to me, I don't have time to test it deeply right now.

File: tests/KafkaFakeTest.php (hunk 1 above)
Comment (gajosu) [reply tests/KafkaFakeTest.php:371]: sure!

Comment (gajosu) [reply tests/KafkaFakeTest.php:371]: sure!

**Review by mateusjunges — APPROVED** [review]

**Pull request merged**